ui/apps/platform/src/Components/ToggledPopoverElement/ToggledPopoverElement.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactElement, ReactNode } from 'react';
```

Comment:
I went back and forth on this – ultimately decided to create the component because it looks like we're using a `button` every time the popover is being used (I assume ADA reasons?) but I'm not hard stuck on this being the right approach and I could easily be swayed both ways